boards/bluerobotics/navigator/init/rc.board_defaults

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# board specific defaults
+#------------------------------------------------------------------------------
+
+# 1K + 4.7K
+param set-default BAT1_V_DIV 5.7
```

Comment:
With the way init is currently handled for these linux boards (posix-configs/rpi/navigator/sub.config) this isn't going to be called by default.

If you wanted to work towards that I'd suggest manually calling it yourself, and you could also add an rc.board_sensors.